test/cli.js

```diff
@@ -59,15 +59,15 @@ describe('cli', function() {
   });
 
   it('should output stylelint warnings', function(done) {
```

Comment:
these are testing other feature, would you mind creating a suite only to test the `importRoot` thing instead of mixing tests up?